test/protobuf/verifier_test.exs

```diff
@@ -0,0 +1,313 @@
+defmodule Protobuf.VerifierTest do
+  use ExUnit.Case, async: true
+
+  alias Protobuf.Verifier
```

Comment:
Can you test the my message proto MyMessage in integration tests as well https://github.com/brexhq/protobuf-elixir/blob/brex-head/test/protobuf/protoc/integration_test.exs. to run the integration test it's `mix test.integration`